Test failure: TypeError: SamVisionLayer.forward() got an unexpected keyword argument 'output_attentions'

Repro: install lightning pose editable from source. At time of writing main is c3a2ad1bb87cd9b15990f00499ef0f7304f9b99a. 
Run pytest

Encounter failures:
```
======================================== short test summary info ========================================
FAILED tests/models/test_base.py::test_representation_shapes_vit - TypeError: SamVisionLayer.forward() got an unexpected keyword argument 'output_attentions'
FAILED tests/models/test_heatmap_tracker.py::test_supervised_heatmap_vit_sam - TypeError: SamVisionLayer.forward() got an unexpected keyword argument 'output_attentions'
FAILED tests/models/test_heatmap_tracker_mhcrnn.py::test_supervised_heatmap_mhcrnn_vit_sam - TypeError: SamVisionLayer.forward() got an unexpected keyword argument 'output_attentions'
FAILED tests/models/test_heatmap_tracker_mhcrnn.py::test_semisupervised_heatmap_mhcrnn_pcasingleview_vit - TypeError: SamVisionLayer.forward() got an unexpected keyword argument 'output_attentions'
```

This is probably a version incompatbility? 
My local version of `transformers` is Version: 4.55.2. 
Axon has version Version: 4.52.4

Testing whether the downgrade fixes it

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -51,7 +51,7 @@ dependencies = [
     "tensorboard (<3.0.0)",
     "torchtyping",
     "torchvision",
-    "transformers",
+    "transformers==4.52.4",
     "typeguard",
     "nvidia-dali-cuda110",
 ]

